lib/addon-sim.js

```diff
@@ -84,8 +87,10 @@ const addon = {
     clear () {},
     isValid () {},
     getTraceSettings (xtrace, localMode) {
+      const traceTaskId = Event.makeFromString(xtrace);
       return {
-        metadata: new Metadata(xtrace),
+        traceTaskId,
```

Comment:
no, javascript has a nice feature (i think since 2015 or 2016) where a variable name in an object is used as the key and the value of the variable is used as the value if no ":" is present. everyone was tired of typing `traceTaskId: traceTaskId`.

Reply:
nice!